lib/babe/babe_integration_test.go

```diff
@@ -50,10 +51,17 @@ var (
 	}
 )
 
-func createTestService(t *testing.T, cfg ServiceConfig) *Service {
+func createTestService(t *testing.T, cfg ServiceConfig, useWestend bool) *Service {
```

Comment:
we should use `westend-dev` or `westend-local` as our spec file and remove the other ones, so the `useWestend` argument is not needed here.

Reply:
I had done this initially to allow for fixing tests to use westend incrementally across multiple PRs, but I have resolved all of them in this one now so the boolean is gone.